cypress/integration/lab.test.js

```diff
@@ -0,0 +1,45 @@
+const deckId = "my-deck-id";
+
+const clickForDeck = (fixture, count = 5) => {
+  cy.intercept(`https://deckofcardsapi.com/api/deck/${deckId}/draw/?count=${count}`, { fixture });
+  cy.get("button").click();
+};
+
+const visitWithFirstDeck = () => {
+  cy.intercept("https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=1", {
+    deck_id: deckId,
+  }).as("newDeck");
+
+  cy.visit("./index.html");
+
+  clickForDeck("cards1.json");
+};
+
+describe("Index", () => {
+  it("starts the select with a default value of 5", () => {
+    cy.visit("./index.html");
+    cy.get("select").should("have.value", 5);
+  });
+
+  it("shows five cards from the retrieved deck ID when the button is clicked", () => {
+    visitWithFirstDeck();
+
+    cy.get(".card")
```

Comment:
[The instructions above](https://github.com/joinpursuit/Pursuit-Core-Web-Async-Await-Lab/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R65) say to use `img` tags with a class of `.image` for each card. So I think either the instructions or the test should be updated so that the specs look for the class and/or element we tell them to use in the instructions. IMO `card` makes more sense as a class name but I really don't have a preference how you reconcile it.